UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentFileShareServiceTest.cs

```diff
@@ -100,5 +101,58 @@ public void WhenInValidRequest_ThenDownloadFileShareServiceFilesReturnsNoFile()
             var result = fulfilmentFileShareService.DownloadFileShareServiceFiles(message, fulfilmentDataResponses, "");
             Assert.IsNotNull(result);
         }
+
+        [Test]
+        public async Task WhenRequestSearchReadMeFilePath_ThenReturnsFilePath()
+        {
+            string batchId = "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272";
+            string exchangeSetRootPath = @"D:\\Downloads";
+
+            A.CallTo(() => fakefileShareService.SearchReadMeFilePath(A<string>.Ignored)).Returns(exchangeSetRootPath);
+            var result = await fulfilmentFileShareService.SearchReadMeFilePath(batchId);
+
+            Assert.IsNotNull(result);
+            Assert.AreEqual("ReadMe Text file path found", "ReadMe Text file path found");
```

Comment:
What is this line doing?

Reply:
This assert statement is incorrect. It should be Assert.AreEqual("ReadMe Text file path found", result); Correct if I am wrong

Reply:
Thanks Janvi

Reply:
Removed Assert.AreEqual as it is not needed. Modified method.